source/dem/localize_contacts.cc

```diff
@@ -31,6 +31,7 @@ localize_contacts(
                        std::tuple<Particles::ParticleIterator<dim>,
                                   Tensor<1, dim>,
                                   Point<dim>,
+                                  unsigned int,
```

Comment:
This is for boundary id? Is there not a type of the boundary id?

Reply:
https://www.dealii.org/developer/doxygen/deal.II/namespacetypes.html#aed8813fee8c8a2edcc6005e6a48c321a

I know they are the same, but it makes readibility better.

Reply:
done